README.md

```diff
@@ -142,6 +144,84 @@ cisco_interface_ospf {"Ethernet1/2 Sample":
 }
 ~~~
 
+## Platform Support
+
+### <a name="provider-platform-support">Provider Support Across Platforms</a>
+
+The following table indicates which providers are supported on each platform. As platforms are added to the support list they may indicate ' `Unsupported`' for some providers that have not completed the test validation process at the time of this release. Some providers will show caveats for a platform if there are limitations on usage, such as with unsupported properties or hardware limitations.
+
+##### Cisco Providers
+
+| ✅ = Supported <br> ❌ = Unsupported | N9k | N30xx | N31xx | N56xx | N6k | N7k | Caveats |
```

Comment:
For the platforms, we should stick to the `Nxxxx` format instead of `N9k` and `N7k` etc...